src/ripple/protocol/impl/SField.cpp

```diff
@@ -281,6 +281,7 @@ CONSTRUCT_TYPED_SFIELD(sfCreateCode,            "CreateCode",           VL,
 CONSTRUCT_TYPED_SFIELD(sfMemoType,              "MemoType",             VL,        12);
 CONSTRUCT_TYPED_SFIELD(sfMemoData,              "MemoData",             VL,        13);
 CONSTRUCT_TYPED_SFIELD(sfMemoFormat,            "MemoFormat",           VL,        14);
+CONSTRUCT_TYPED_SFIELD(sfData,                  "Data",                 VL,        15);
```

Comment:
You may want to consider using a non-common code here? On the other hand `Data` is generic enough that it may be useful elsewhere, so... maybe it's fine.

Reply:
This was resolved but I forget which commit.